### PR TITLE
Config2 - New version using new proposed config parser

### PR DESCRIPTION
**Pull request by otaviobp**
New version using config parser proposed in https://github.com/01org/mavlink-router/pull/65

### CODE REVIEW

File: a/main.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[gstreamer]
+muxer=rtph264pay
+encoder=vaapih264enc
+converter=autovideoconvert
+[v4l2]
+blacklist=video0 video1 video2 video3 video4 video5 video6 video7 video8 video9 video10 video11 video12
```
Comment (mbelluzzo) [inline a/main.conf]: That is long list hehehe. Just for curiosity, why is that needed?

File: src/conf_file.h [hunk 2]
```diff
@@ -0,0 +1,139 @@
+/*
+ * This file is part of the MAVLink Router project
```
Comment (mbelluzzo) [inline src/conf_file.h]: Is this intended to stay this way?

**Comment by otaviobp** [comment]
merged
